Source/WebCore/platform/graphics/mac/controls/ControlMac.mm

```diff
@@ -325,7 +367,6 @@ static void drawCellFocusRing(const FloatRect& rect, NSCell *cell, NSView *view)
         return;
 
     // We can't paint an NSComboBoxCell since they are not height-resizable.
-    LocalDefaultSystemAppearance localAppearance(style.states.contains(ControlStyle::State::DarkAppearance), style.accentColor);
```

Comment:
Note: I think this is safe to remove, since the callers of `drawListButton` set the system appearance.

Reply:
Yes this is right.